src/arch/x86_64/mm/paging.rs

```diff
@@ -64,6 +70,16 @@ unsafe fn recursive_page_table() -> RecursivePageTable<'static> {
 	}
 }
 
+unsafe fn identity_mapped_page_table() -> OffsetPageTable<'static> {
```

Comment:
I'd appreciate a short doc comment, as I think it is not 100% obvious what the function is doing.